examples/electron/package.json

```diff
@@ -33,6 +34,8 @@
   "devDependencies": {
     "@types/chai": "^3.5.1",
     "@types/webdriverio": "^4.7.0",
+    "@types/yargs": "^6.6.0",
```

Comment:
I thought that was only true for theia-core since it was consumed by the examples... 
Why is it true for the examples?